www/caddy/src/opnsense/mvc/app/controllers/OPNsense/Caddy/forms/general.xml

```diff
@@ -33,6 +33,12 @@
             <type>checkbox</type>
             <help><![CDATA[Run this service as "www" user and group, instead of "root". This setting increases security, but comes with the hard restriction that the well-known port range can not be used anymore. After enabling and saving this setting, the service has to be totally restarted. For this, please disable Caddy and press Apply. Afterwards enable Caddy and press Apply. This setting is reversible by following the same steps.]]></help>
         </field>
+        <field>
+            <id>caddy.general.DisableQuic</id>
```

Comment:
Generally speaking an „enable quic“ is better UI. You end up negating the logic and making mistakes by introducing inverted flags.

Reply:
But its an option that is on by default. And isnt it better UI to allow the user to revert back to defaults by emptying all fields and unchecking all checkboxes?

If some checkboxes are 1 default, and some are 0 default, which will be the default? Sounds confusing to me.

Reply:
https://github.com/opnsense/plugins/pull/4077#issuecomment-2214389184

Reply:
the defaults are not set by unchecking boxes

if you remove the full caddy config the migration will set the actual defaults for each setting.

Reply:
this is also a problem in support: did you enable quic? did you enable the disable for quic? did you disable quic? you just want to avoid these ambiguities as much as possible

Reply:
I can do a selectpicker instead that lets people choose the protocols they want, and do a blancdesc that is the default. Kinda like here in the backend listeners:

https://github.com/opnsense/plugins/blob/ea150f2e901cb019d33645998c401ef0659cde6b/www/caddy/src/opnsense/mvc/app/models/OPNsense/Caddy/Caddy.xml#L314-L321

Reply:
Also fine if you prefer that.

Reply:
Ill do the selectpicker so its the same in front and backend. Thanks so much for the feedback. :)